cmd/rook/main.go

```diff
@@ -87,11 +87,6 @@ func init() {
 	// load the environment variables
 	flags.SetFlagsFromEnv(rootCmd.Flags(), RookEnvVarPrefix)
 	flags.SetFlagsFromEnv(rootCmd.PersistentFlags(), RookEnvVarPrefix)
-
-	rootCmd.Run = func(cmd *cobra.Command, args []string) {
```

Comment:
so if someone runs the rook binary directly with no args, what will the experience be now?

Reply:
It will just print the command line args. Not sure why I removed this though, I'll put it back. It's a helpful message.

Reply:
actually, in the long term we don't want this message. Do you think we need this message in 0.6? My thought was that the past month of this message in master was sufficient

Reply:
sure i am OK with removing the message now, especially since all the documentation and everything is gone and we haven't heard from anyone about missing it.